plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -344,3 +351,44 @@ def chunks(batch: List[Any], n: Optional[int]) -> Iterable[List[Any]]:
             raise ValueError("n must be an integer greater than 0")
         for i in range(0, len(batch), n):
             yield batch[i : i + n]
+
+
+def process_results(
```

Comment:
what is process_results doing?
what results is it processing? and why?

Reply:
The results (best params) could have a "." in the key. That is converted to "_".